### DISCUSSION TITLE
Claim all blocks in an area except for certain (specified) blocks.

### DISCUSSION THREAD

**Discussion by MilkMGN** [GriefPrevention/GriefPrevention#1271] [General]
Sorry if this is a blatantly obvious featur, but im making a shop area on my server and need to no allow areas such as the pathway to be claimed. doing this manually seems tedious, so i was wanting to see if this can be added/is added.

**Comment by Jikoo** [comment]
I don't see how you wouldn't do it manually either way without then preventing other random areas from being claimed if users built with certain blocks.

As far as your options go...
1) Create claims along the paths
2) Create one large claim over the entire shop and give users trust in subclaims (though [it looks like this has to be manually managed](https://github.com/RoboMWM/GPRealEstate/blob/ccb60f6733cec0b3e0239388cbc855ca3f3094e0/src/main/java/me/SuperPyroManiac/GPR/GPREListener.java#L450) - you'd have to add trust yourself)
